testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -11732,7 +11733,8 @@ void assertSubFunReturns(boolean binary, String s, int start,
       f.checkScalar("least(CAST(NULL AS INTEGER), CAST(NULL AS INTEGER))", isNullValue(),
           "INTEGER");
     };
-    final List<SqlLibrary> libraries = list(SqlLibrary.POSTGRESQL, SqlLibrary.REDSHIFT);
+    final List<SqlLibrary> libraries =
+        list(SqlLibrary.POSTGRESQL, SqlLibrary.REDSHIFT, SqlLibrary.SPARK);
```

Comment:
Why `SqlLibrary.REDSHIFT` works? I noticed before this PR, it already has it.

Reply:
It inherits functions from PG
```
// SqlLibrary.java
REDSHIFT("r", "redshift", POSTGRESQL)
```